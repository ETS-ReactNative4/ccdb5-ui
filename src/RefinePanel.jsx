import React from 'react';
import './RefinePanel.less';

export default class RefinePanel extends React.Component {
  render() {
    return (
      <section className="refine-panel">
        <h3>Filter results by...</h3>
      </section>
    );
  }
}
