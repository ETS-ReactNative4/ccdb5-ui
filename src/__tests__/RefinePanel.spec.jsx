import React from 'react';
import RefinePanel from '../RefinePanel';
import renderer from 'react-test-renderer';

describe('initial state', () => {
  it('renders without crashing', () => {
    const target = renderer.create(
      <RefinePanel />
    );

    let tree = target.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

