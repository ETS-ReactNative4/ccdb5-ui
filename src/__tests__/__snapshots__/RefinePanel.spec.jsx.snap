exports[`initial state renders without crashing 1`] = `
<section
  className="refine-panel">
  <h3>
    Filter results by...
  </h3>
</section>
`;
